### PR TITLE
finish process lesson

### PR DESCRIPTION
**Pull request by minh93**

### CODE REVIEW

File: app/views/categories/_category.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-md-4">
+    <%= category.name %>
+  </div>
+  <div class="col-md-8">
+    <%= category.description %>
+    <blockquote>
+      <% category.words.each do |w| %>
+      <%= w.content %>
```
Comment (yuyuvn) [inline app/views/categories/_category.html.erb]: indent

File: config/routes.rb [hunk 2]
```diff
@@ -6,4 +6,5 @@
   delete "logout" => "sessions#destroy"
   resources :users
   resources :relationships, only: [:create, :destroy]
+  resources :categories
```
Comment (yuyuvn) [inline config/routes.rb]: filter

File: app/views/categories/_category.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-md-4">
+    <%= category.name %>
+  </div>
+  <div class="col-md-8">
+    <%= category.description %>
+    <blockquote>
+      <% category.words.each do |w| %>
```
Comment (phuongbk) [inline app/views/categories/_category.html.erb]: change w to word

File: app/controllers/categories_controller.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class CategoriesController < ApplicationController
+  def index
+    @categories = Category.all
```
Comment (dieunb) [inline app/controllers/categories_controller.rb]: paginate

**Pull request merged**